### ISSUE TITLE
Randomize Karamenbo starting height

### ISSUE THREAD

**Issue by slashinfty** [slashinfty/sml2randomizer#8]
For values `[0xE9D6, 0xE9D9, 0xE9DF, 0xE9E2, 0xE9E5]`, assign a random byte between `0x34` and `0x3E` using `nextInt(0xB)+0x34`. Push at conclusion of season 3 of randomizer league.

**Comment by slashinfty** [comment]
Look into starting heights of rising, falling, and skull platforms in Wario's Castle (only for Beast Mode).

```javascript
for(var i = 0xE9A3; i < 0xE9CE; i += 3) {
rom[i] = nextInt(some value) + another value;
}
```

**Comment by slashinfty** [comment]
Implemented with v2.02
